app/sessionStore.js

```diff
@@ -428,6 +429,17 @@ module.exports.cleanAppData = (immutableData, isShutdown) => {
     immutableData = immutableData.deleteIn(['ledger', 'locations'])
   }
 
+  try {
+    const basePath = getStoragePath('ledger-favicons')
+    const fs = require('fs')
+    if (!fs.existsSync(basePath)) {
```

Comment:
should use the async versions of these to avoid blocking the UI thread

Reply:
it needs to be synchnrous because this directory must exist before `cleanFavicons` is called, and `cleanFavicons` returns immutableData synchronously. I could add a migration flag that indicates whether this directory was created, which would avoid having to do the synchronous mkdir in future runs.